Sources/Build/BuildManifest/LLBuildManifestBuilder+Product.swift

```diff
@@ -33,13 +33,19 @@ extension LLBuildManifestBuilder {
             testInputs = []
         }
 
+        // Create a phony node to represent the entire target.
+        let targetName = try buildProduct.product.getLLBuildTargetName(config: self.buildConfig)
```

Comment:
please check if this has performance implications, I vaguely remeber @neonichu ran into something like that with this API, tho no 100% sure

Reply:
This phony node is not new, it's only moved from previous L62 where it was declared too late for us. I had to move it up to this line.